libwebauthn/src/transport/hid/channel.rs

```diff
@@ -443,3 +447,26 @@ bitflags! {
         const NO_MSG = 0x08;
     }
 }
+
+impl Ctap2AuthTokenStore for HidChannel<'_> {
+    fn store_uv_auth_token(
+        &mut self,
+        identifier: Ctap2AuthTokenIdentifier,
+        pin_uv_auth_token: &[u8],
+    ) {
+        self.auth_token_storage = Some((identifier, pin_uv_auth_token.to_vec()));
+    }
+
+    fn get_uv_auth_token(&self, identifier: &Ctap2AuthTokenIdentifier) -> Option<&[u8]> {
+        if let Some((id, token)) = &self.auth_token_storage {
+            if id.is_satisfied_by(identifier) {
```

Comment:
I would read this as meaning `id` is satisfied by existing permissions `identifier`, but I think this means the opposite.